validator/src/cli.rs

```diff
@@ -372,8 +372,9 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                        [default: ask --entrypoint, or 127.0.0.1 when --entrypoint is not provided]"),
         )
         .arg(
-            Arg::with_name("tpu_host_addr")
-                .long("tpu-host-addr")
+            Arg::with_name("public_tpu_addr")
+                .long("public-tpu-address")
```

Comment:
for consistency, can the long one also be `-addr` instead of `-address`?

Reply:
In the UI `address` is used almost everywhere:
```
% cat validator/src/cli.rs | grep -e "long.*addr" -B 1
            Arg::with_name("rpc_faucet_addr")
                .long("rpc-faucet-address")
--
            Arg::with_name("public_tpu_addr")
                .long("public-tpu-address")
--
            Arg::with_name("public_rpc_addr")
                .long("public-rpc-address")
--
            Arg::with_name("bind_address")
                .long("bind-address")
--
            Arg::with_name("rpc_bind_address")
                .long("rpc-bind-address")
--
            Arg::with_name("accountsdb_repl_bind_address")
                .long("accountsdb-repl-bind-address")
--
            Arg::with_name("allow_private_addr")
                .long("allow-private-addr")
--
            Arg::with_name("bind_address")
                .long("bind-address")
```